runtime/compiler/env/ClassLoaderTable.cpp

```diff
@@ -70,8 +70,12 @@ int32_t TR_PersistentClassLoaderTable::_numClassLoaders = 0;
 void
 TR_PersistentClassLoaderTable::associateClassLoaderWithClass(void *classLoaderPointer, TR_OpaqueClassBlock *clazz)
    {
+   //fprintf(stderr, "PLCT: associateClassLoaderWithClass %p %p\n", classLoaderPointer, clazz);
```

Comment:
Suggestion for the future - wrap debug code in an easy to toggle ifdef rather than leaving it comment out.  Much easier to work with if it's a single thing to enable rather than many separate lines to uncomment